sql/_30_banana_pie_qa/issue_20523_foreign_key/cases/20523_01.sql

```diff
@@ -0,0 +1,46 @@
+-- TEST CASE For:
+-- [CBRD-20523] Foreign Key Referential Constraints check on INSERT
+-- Notice:
+-- We changed the rule to check 'Foreign Key Referential Constraints' on INSERT/UPDATE
+-- So, apply of this test case to former CUBRID will lead to verdict 'fail'
+--
+-- 
+-- CREATE two TABLEs, Parent/Child Respectively
+-- 
+-- 1. CREATE Parent TABLE
+CREATE TABLE a_tbl (
```

Comment:
To avoid a conflict with other cases, we have kept a rule that `CREATE TABLE` follows `DROP TABLE IF EXIST`.

```sql
DROP TABLE IF EXISTS a_tbl;
CREATE TABLE a_tbl (
  ...
```